testing/upload_validation.ts

```diff
@@ -1114,4 +1115,36 @@ const legacyPackMetadataSchema = validateFormulas(
         "Specify the domain that your pack makes http requests to using `networkDomains: ['example.com']`",
       path: ['networkDomains'],
     },
-  );
+  )
+  .superRefine((data, context) => {
+    if (!data.defaultAuthentication) {
+      return;
+    }
+    // Pack has multiple network domains and uses authentication.
+    if (data.defaultAuthentication.requiresEndpointUrl || data.defaultAuthentication.type === AuthenticationType.None) {
+      // We're ok if there's a user-supplied endpoint domain if auth is just being disabled explicitly.
```

Comment:
I think there's a missing "or".

Reply:
Simplified this comment because switching from `any` to `PackVersionMetadata` required me to check for `None` earlier anyway.